### PR TITLE
feature to filter out desired labels

### PR DESCRIPTION
**Pull request by IanMenendez**
This feature aims to make it easier to get desired values from a  cluster

I got this idea since sometimes I wanted to select a big cluster of labels but only get the ones that where a minority

### CODE REVIEW

**Review by JulesBelveze — CHANGES_REQUESTED** [review]
Cool stuff! 
Could you make sure that the `MultiChoice` still works if the `color` column contains int? Also, `color` can be of type `float`. I don't know how this would behave in such case..

For example with `color` being an int I get the following issue:
```
ValueError: failed to validate MultiChoice(id='1058', ...).options: expected an element of List(Either(String, Tuple(String, String))), got seq with invalid items [0, 1, 2]
```

**Comment by IanMenendez** [comment]
not the prettiest solution. 
I converted the color column to strings, as this is the type that MultiChoice accepts.
This works because when creating the new tab the only values that you use are the project id and the id of the label

File: bundler/text.py [hunk 1]
```diff
@@ -61,7 +69,11 @@ def save():
         tab_btn = Button(label="Create tab")
         tab_btn.on_click(save)
 
-        controls = column(p, tab_name, tab_btn)
+        df['color'] = df['color'].astype(str) #MultiChoice works only with Strings
+
```
Comment (JulesBelveze) [inline bundler/text.py:73]: That's gonna be quite nasty if `color` is of type `float`.
I would either disable this feature in this case or use some sort of cursor (like [here](https://demo.bokeh.org/movies))

**Review by JulesBelveze — CHANGES_REQUESTED** [review]

**Review by JulesBelveze — APPROVED** [review]

**Comment by JulesBelveze** [comment]
@IanMenendez LGTM, I took the freedom to refactor a bit. Feel free to merge 😃

**Comment by IanMenendez** [comment]
> @IanMenendez LGTM, I took the freedom to refactor a bit. Feel free to merge 😃

great! 
Can't merge because I don't have write access to  the repo  :sweat_smile:

**Pull request merged**